routes/common/load_balancer_helper.rb

```diff
@@ -66,7 +105,7 @@ def get
       vms = @resource.private_subnet.vms_dataset.authorized(@user.id, "Vm:view").all
       attached_vm_ids = @resource.vms.map(&:id)
       @app.instance_variable_set(:@attachable_vms, Serializers::Vm.serialize(vms.reject { attached_vm_ids.include?(_1.id) }))
-      @app.instance_variable_set(:@lb, Serializers::LoadBalancer.serialize(@resource, {detailed: true}))
+      @app.instance_variable_set(:@lb, Serializers::LoadBalancer.serialize(@resource, {detailed: true, vms_serialized: true}))
```

Comment:
what do we need serialized vms for?

Reply:
we are using location, vm name and the load balancer health check state of the vm. Instead of returning a custom map, I decided to serialize the whole VM. Those are not needed for APIs but just the Web endpoint.